### DISCUSSION TITLE
help/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#612] [Announcements]
# help/

星空组网使用步骤 一、安装客户端 (/quick/); 二、通过虚拟IP访问 (/case/); 关注微信公众号获取更多有用信息。; 扫一扫使用微信小程序管理。;

https://ip4.ink/help/

**Comment by 912988434** [comment]
功能上有一些小建议
1、建议控制台上可以加展示客户端信息，比如每个客户端流量统计，上次上下线时间等
2、服务端可以暴露metrics，方便用户灵活监控，可以方便使用第三方工具比如 grafana 更好的进行展示
3、 增加告警功能比如客户端下线通知等可以接入常用的一些渠道比如钉钉，飞书，微信，webhook等
